frontend/src/components/DefenceBox/DefenceConfigurationRadioButton.css

```diff
@@ -1,10 +1,13 @@
 .defence-radio-button {
 	position: relative;
 	display: inline-block;
+	display: flex;
+	margin: 0.1rem;
```

Comment:
Remove this, seems unnecessary. We already have `gap: 0.5rem` on the parent element, so if you want more vertical space between radios, bump that up a little.

Also, I'd prefer `gap: 0.75rem` on this class, instead of margin-left on `.defence-radio-button span` below.